examples/plot_h2_grid.py

```diff
@@ -0,0 +1,47 @@
+"""
```

Comment:
rename the file for consistency with the current file namings: plot_grid_h2 (what we plot, THEN, on which space we plot it).